indicator_templates/Enumerations/PopulationEnumeratedByArea.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+namespace App\IndicatorTemplates\Enumerations;
+
+use App\Http\Livewire\Chart;
+use App\Services\Interfaces\BarChart;
+use Carbon\Carbon;
+use Illuminate\Support\Collection;
+
+/**
+ * Population enumerated by area
+ * A BarChart showing the population enumerated by area, with a line chart showing the target population to be enumerated.
+ *
+ * @category BarChart
+ */
+abstract class PopulationEnumeratedByArea extends Chart implements BarChart
+{
+    public const POPULATION_DATA_KEY =[ 'area_code', 'population'];
```

Comment:
I suggest you make this a generic (and the same across all templates) constant name so that we might be able to use it in the "Tester" concept we discussed.